template/views/authentication/server.js

```diff
@@ -4,5 +4,5 @@ const app = require("./app");
 const PORT = process.env.PORT || 3000;
 
 app.listen(PORT, () => {
-  console.log(`Server listening on port http://localhost:${PORT}`);
-});
+  console.log(`Server listening on port ${PORT}`);
```

Comment:
having the `http://localhost:${PORT}` allows to "cmd+click" on the terminal and open in the browser. should it be removed?

Reply:
Yes sure, we can revert back to the old message.